pyiron/atomistics/structure/factory.py

```diff
@@ -0,0 +1,333 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+from ase.build import bulk
+from ase.build import (
+        add_adsorbate,
+        add_vacuum,
+        bcc100,
+        bcc110,
+        bcc111,
+        diamond100,
+        diamond111,
+        fcc100,
+        fcc110,
+        fcc111,
+        fcc211,
+        hcp0001,
+        hcp10m10,
+        mx2,
+        hcp0001_root,
+        fcc111_root,
+        bcc111_root,
+        root_surface,
+        root_surface_analysis,
+        surface as ase_surf,
+    )
+from ase.io import read
+import numpy as np
+from pyiron.atomistics.structure.pyironase import publication as publication_ase
+from pyiron.atomistics.structure.atoms import CrystalStructure, ase_to_pyiron, Atoms
+from pyiron.atomistics.structure.periodic_table import PeriodicTable
+from pyiron_base import Settings
+import types
+
+__author__ = "Sudarsan Surendralal"
+__copyright__ = (
+    "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Sudarsan Surendralal"
+__email__ = "surendralal@mpie.de"
+__status__ = "production"
+__date__ = "May 1, 2020"
+
+s = Settings()
+
+
+class StructureFactory:
+    def ase_read(self, *args, **kwargs):
```

Comment:
```suggestion
    @staticmethod
    def ase_read(*args, **kwargs):
```

Reply:
I know it looks like this one can be static, but actually the line below it (`ase_read.__doc__ = read.__doc__`) only works when it's got `self`. I can kind of guess why, but it surprised me.